MeetupPlatformApi/Swagger/SwaggerDocConfiguration/SwaggerSwashbuckleConfiguration.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace MeetupPlatformApi.Swagger.SwaggerDocConfiguration;
```

Comment:
Do we really need such deeply nested namespace? Inside `Swagger` namespace we only have the `SwaggerDocConfiguration`. Can't we just skip the `SwaggerDocConfiguration` part then?